.github/workflows/external-integrations-unit-tests.yml

```
@@ -0,0 +1,41 @@
+name: EXTERNAL INTEGRATIONS unit tests
```

Just to be consistent with the naming of the rest of the workflows

```suggestion
name: External integrations unit tests
```